cmd/open.go

```diff
@@ -0,0 +1,52 @@
+package cmd
+
+import (
+	"fmt"
+	"os/exec"
+	"runtime"
+	"strings"
+
+	"github.com/CircleCI-Public/circleci-cli/git"
+	"github.com/spf13/cobra"
+)
+
+func openBrowser(url string) error {
```

Comment:
Perhaps this? https://godoc.org/github.com/pkg/browser

A quick skim of the code suggests it's not obviously better, but `pkg` is generally a trusted source

Reply:
good idea